source/Libraries/ItchioLibrary/ItchioLibrarySettingsViewModel.cs

```diff
@@ -45,6 +56,26 @@ public RelayCommand<object> LoginCommand
             });
         }
 
+        public RelayCommand<GameClassificationItem> ToggleGameClassificationCommand

```

Comment:
I think it would be better to just have a dictionary with with all game types and false/true for whether they are enabled or not. Then you can get rid of all these command and bind directly to selected state.

Reply:
I tried the dictionary, but it's not possible to use a two-way binding with value unless you implement a whole ObservableDictionary class, however I was able to get rid of the command by populating the list with all the wrapped enum values.

Another thing I noticed is that the previous saved list get appended to the default one when deserialization happens, so I had to move the initialization into the constructor to avoid duplicate elements.

Reply:
`ObservableConcurrentDictionary` is available in the SDK. As far as list being duplicated, add this attribute: `[JsonProperty(ObjectCreationHandling = ObjectCreationHandling.Replace)]` to the property. You will need to [add json.net dependency](https://playnite.link/docs/master/tutorials/extensions/plugins.html#plugin-dependencies).

Reply:
I was able to replace the collection with the dictionary, but I had to go back to use a command because the `Value` in `KeyValuePair` is read only; howerver, it resolved the problem with duplicates value, so there's no need for the json.net attribute.

I found that the syntax `{Binding Path=[game]}` works too, but it requires to make all the checkboxes manually as I could not find a way to have a dynamic value inside Path (`game` gets interpreted as a literal value).